pkg/controller/plan/validation.go

```diff
@@ -491,6 +514,9 @@ func (r *Reconciler) validateVM(plan *api.Plan) error {
 	if len(multiplePodNetworkMappings.Items) > 0 {
 		plan.Status.SetCondition(multiplePodNetworkMappings)
 	}
+	if len(staticStorage.Items) > 0 {
+		plan.Status.SetCondition(staticStorage)
```

Comment:
but that just means that there's at least one pv that was created statically, right?
would checking the 'provisioner' field of the storage class would make sense?

Reply:
I think `provisioner` is a mandatory field so it would be difficult to use it for this

Reply:
it checks per vm, but i didn't verify it yet

Reply:
It does checks it per VM, the plan will be in critical status. On the migration you will be able to see on which VM ID and name you hit the rule.

About the provisioner, we said - no provisioner is ok. but i am not sure it covers all the cases. in such case i think using the current implementation also covers the no provisioner case.